CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+cmake_minimum_required(VERSION 3.20...3.29)
+# 3.20 is the minimum requirement for LLVM
+# https://www.llvm.org/docs/CMake.html#quick-start
+
+project(
+        ${SKBUILD_PROJECT_NAME}
+	VERSION ${SKBUILD_PROJECT_VERSION}
+	LANGUAGES CXX)
+
+set(
+	CMAKE_CXX_STANDARD
+	14
+	CACHE STRING "The C++ standard to compile with, 14+")
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
```

Comment:
```suggestion

# Linking to OpenMP will be outside the source tree, so include it in INSTALL_RPATH anyway
# Repair steps will bundle it as needed.
set(CMAKE_INSTALL_RPATH_USE_LINK_PATH ON)

```

This is @henryiii's suggestion from https://github.com/j-s-ashley/Wasserstein/pull/5#issuecomment-2274221318